packages/volto/src/components/index.js

```diff
@@ -22,6 +22,7 @@ export { default as Title } from '@plone/volto/components/theme/Title/Title';
 export { default as DefaultView } from '@plone/volto/components/theme/View/DefaultView';
 export { default as Pagination } from '@plone/volto/components/theme/Pagination/Pagination';
 export { default as Tags } from '@plone/volto/components/theme/Tags/Tags';
+export { default as RelatedItems } from '@plone/volto/components/theme/RelatedItems/RelatedItems';
```

Comment:
Let's not add new components to the barrel files, since we want to remove them (#6507)
```suggestion
```